docs_src/general/EdgeX_CN.md

```diff
@@ -0,0 +1,153 @@
+
+**Observable**
+
+Perhaps one of the greatest contributions of the CNCF community to CN computing is providing all sorts of tools and technologies to observe and analyze CN applications in the cloud.  Tools like [Prometheus](https://prometheus.io/) make monitoring CN containers and their resource utilization a breeze.  EdgeX does not come with native observability capabilities.  When using EdgeX containers tools like Prometheus for observability and analytics can be used to monitor EdgeX services.  Likewise, on some platforms and OS, there are ingredients (like Linux process status or system monitor for snaps) that can be used to help facilitate some level of monitoring.  But these are not provided by EdgeX, usually require additional work by an adopter, and may not provide the level of inspection detail required.  EdgeX is, with the Kamakura release, starting to provide more system level data (versus sensor data), metrics and events via message bus that an adopter can subscribe to in order to do more observing/analyzing of the EdgeX services.  This, however is raw data, to which some additional tooling will be required to provide either human or machine monitoring of the data on top to make sense of it.
+
+## The Future of Cloud Native and EdgeX
+As cloud native computing technology and principals expands to more levels of our software realms and as the edge begins to become more indistinguishable from any other part of our computing network, it is inevitable that EdgeX will become more CN like.  Or perhaps put more precisely, cloud native and edge native are tending toward each other.  Edge computing environments are becoming less resource constrained in many places.  The CNCF is looking to bring CN technology and tools (like Kubernetes) to the edge.  Additionally, there are places where EdgeX improvements can help to bridge the CN | EN divide. 
+
+**Kubernetes Support**
+
+As lighter weight Kubernetes infrastructure becomes available (e.g. K3s, KubeEdge, Minikube, etc. - see a [comparison](https://www.itprotoday.com/cloud-computing-and-edge-computing/lightweight-kubernetes-showdown-minikube-vs-k3s-vs-microk8s) for context) and are improved upon, and/or as more edge computing environments get more resources, one of the chief CN technologies - that is Kubernetes - or its close cousin will emerge to better facilitate deployment, orchestration, and monitoring (observability) of container based workloads at the edge.  EdgeX must be prepared to support and embrace it as it has containers and snaps - yet still recognize that the lowest common denominator of edge platforms may only support bare metal type deployments for the foreseeable future.
```

Comment:
> the lowest common denominator of edge platforms may only support bare metal type deployments for the foreseeable future.

"Bare metal" must means no hypervisor.  Did you really mean to say "standard processes"?

I think a lot of this document revolves, either directly or indirectly, on the capabilities that EdgeX can offer when running on the minimal platform.  Can this document be made more succinct if you lead with that?

Reply:
I clarified the bare metal reference.  I agree with the idea that "designed to run on minimal platform"  says a lot.  I added a couple of sentences right at the top to suggest this.  That alone may still leave questions in CNCF follwers.  So I still think the document as is (while long) addresses a lot of our rationale and reasons for not being a cloud native app.